sentences/sv/homeassistant_HassTurnOn.yaml

```diff
@@ -7,20 +7,3 @@ intents:
       - sentences:
           - <öppna_gardiner> <name>
         response: cover
```

Comment:
Missed this one. Is it because it don't have slots?

Reply:
Yup. And what's worse, it could provide that reply in case you ask it to turn on something other than a cover. You should add `requires_context`